x-pack/plugin/sql/sql-cli/src/main/java/org/elasticsearch/xpack/sql/cli/Cli.java

```diff
@@ -102,6 +103,11 @@ public Cli(CliTerminal cliTerminal) {
             .withRequiredArg()
             .ofType(Boolean.class)
             .defaultsTo(Boolean.parseBoolean(System.getProperty("cli.check", "true")));
+        this.lenientOption = parser.acceptsAll(
```

Comment:
Looking at the already existent commands and options for CLI, I would have inclined to make `lenient` a command and not a cli option. For example, `fetch_size` is a command (`FetchSizeCliCommand` class). The command can be executed multiple times during the life of the cli and an user can change the leniency with each query executed.

Reply:
From the session configuration point of view, this would make indeed sense.
From a practical POV, not sure if changing this behavior on a query-by-query case would be as useful, but for those users that use the CLI integrated into other scripts, having it as an argument might be preferred.

Reply:
I agree, having a command would make it much more flexible, I'm implementing it now.
Do you think it makes sense to keep both (ie. the command AND the cli option) or should we just have the command?

Reply:
It doesn't make sense to have it in both places. We are not doing this for any setting/config so far. Please, implement it as a command only.

Reply:
OK, I removed the CLI option and replaced it with a command for now (`lenient = true/false`).
In case we decide to go with both, re-enabling the option is straight forward.